src/components/ForgotPassword/ForgotPassword.js

```diff
@@ -75,11 +75,16 @@ const ForgotPassword = ({ intl }) => {
           </Form>
         </Formik>
         <footer>
-          <FormattedHTMLMessage
-            tagName="small"
-            id="signup.copyright_HTML"
-            values={{ year: new Date().getFullYear() }}
-          />
+          <p>
+            <FormattedHTMLMessage tagName="small" id="common.recaptcha_legal_HTML" />
```

Comment:
Como hablabamos en el chat:

> A mi modo de ver, me parece que lo mejor es que el texto este en ese tamaño, pero en color #cccccc y los links con el mismo estilo de la fuente texto… si la persona se posa sobre los links va a ver el cursor pointer… pero me parece que no deberia visualizarse como link, porque realmente no nos interesa que el usuario confunda esos links con los nuestros.

Reply:
Me parece bien, voy a trabajar junto con @GusBaamonde para aplicar los cambios que me sugerís.

¡Gracias por el feedback!

Reply:
@santiagopaolucci y @andresmoschini Estoy de acuerdo con lo que resolvieron. 

Lo que propongo en los casos de LOGIN y RECUPERO DE CONTRASEÑA es resumir un poco el contenido para que se note todavía menos en español: Sitio protegido por reCAPTCHA. Política de Privacidad y Condiciones del Servicio de Google.

En el caso de SIGN UP pienso que podríamos sumar a Google entre las herramientas que mencionamos en la sección de Destinatarios del footer RGPD, de esta manera: 

ESPAÑOL: Destinatarios: Tus datos serán guardados por Doppler, Zoho como CRM, Google como proveedor del servicio de reCAPTCHA, Digital Ocean, Cogeco Peer1 y Rackspace como empresas de hosting.

INGLÉS: Recipients: Your data will be saved by Doppler, Zoho as CRM, Google as the provider of reCAPTCHA service, Digital Ocean, Cogeco Peer1 and Rackspace as hosting companies.

Reply:
Buenisimo, creo una tarea para esto!

Muchas gracias Caro!